challenge.rb

```diff
@@ -1,26 +1,25 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{ |x| x.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.reject { |e| e != "dog"  }
```

Comment:
I think this returns an array with one element (the "dog"), but I wouldn't expect the result to be an array.

Reply:
Oops. I was wrong. Ignore that ^

Completely different nitpick: `reject` and `!=` are both "negative" in a sense, making this a double negative in my opinion. Can this be rephrased positively?